src/main/java/eu/dissco/core/translator/maven/MavenRunner.java

```diff
@@ -0,0 +1,63 @@
+package eu.dissco.core.translator.maven;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MavenRunner {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MavenRunner.class);
```

Comment:
Why not use slf4j?

Reply:
This is not a bean and doesn't use Spring. So we need to be a bit old-fashioned and declare stuff ourselves :wink: It is just a standalone piece of code with a simple main function.